string buildingName
string floorName
float64 x
float64 y
float64 yaw

